README.md

```diff
@@ -30,13 +30,13 @@ Please check the [CLI Reference](docs/sbom-tool-cli-reference.md) document for a
 
 ##### WinGet
 
-```sh
+```powershell
 winget install Microsoft.SbomTool
 ```
 
 ##### Homebrew
 
-```bash
+```shell
```

Comment:
Visually identical, and bash is appropriate for any homebrew environment (see https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md)